cdk/README.md

```diff
@@ -58,7 +58,7 @@ The app consists of **three** main CDK stacks:
 ### Installation
 ```bash
 git clone <repository-url>
```

Comment:
The top-level README.md should also briefly mention installing ACM4NE with CDK and include a link to point this README file.